docs/js/class_hex.js

```diff
@@ -458,46 +458,9 @@ class Puzzle_hex extends Puzzle {
         }
     }
 
-    draw_point() {

```

Comment:
I can not see from where all the draw_point() methods are used. If you think this method is not legacy code, I can revert this changes and replace the colors accordingly.

Reply:
Yeah, I also didn't find any use for it. Probably just carried over from the old version. We can delete it for now.